content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -948,6 +948,87 @@ the corresponding layout and fiscal stamps stated in the :ref:`Invoice printed r
 .. seealso::
    :doc:`Client account creation <../../websites/ecommerce/checkout_payment_shipping/checkout>`
 
+Liquidity product direct sales
+------------------------------
+
+The liquidity product direct sale is used in cases where sales are made to a third party. In these
+cases, the seller and the proprietary company of the goods can register their corresponding sales
+and purchase. This feature is included when :ref:`installing <general/install>` the *Argentinian
+Electronic Invoicing* module (`l10n_ar_edi`).
+
+Configuration
+~~~~~~~~~~~~~
+
+Document types
+**************
+
+This feature includes the addition of the following document types: 23, 24, 25, 26, 27, 28, 33, 43,
+45, 46, 48, 58, 60, 61, 150, 151, 157, 158, 161, 162, 164, 166, 167, 171, 172, 180, 182, 186, 188,
+332.
+
+.. important::
+   Note that by default these document types are *inactive*. For the workflow to work, it is
+   necessary to *activate* these documents.
+
+To activate them, navigate to :menuselection:`Accounting --> Configuration --> Document Types` and
+remove any active filters. Then, find the documents and activate them by toggling the option
+:guilabel:`Active`.
+
+Purchase journal
+****************
+
+A purchase journal is needed to generate an electronic vendor bill with a document type *Liquidity
+Product*. This journal needs to have an active synchronization with the AFIP as this electronic
+document will be sent to the AFIP.
+
+To modify the existing purchase journal, or create a new purchase journal, navigate to
+:menuselection:`Accounting --> Configuration --> Journals`. Then, click the :guilabel:`New` button,
+and fill in the following required information:
+
+- :guilabel:`Type`: select :guilabel:`Purchase` from the drop-down menu for vendor bill journals.
+- :guilabel:`Use Documents`: check this field for the journal to select the electronic document type
+  (in this case *sales account* and *liquid product*).
```

Comment:
I would separate the instruction (select X) from the additional info on what it does exactly, especially when it's not relevant to get the configuration right. It's easier to follow that way.